com.microsoft.mrtk.graphicstools.unity/Editor/ShaderGUIs/StandardShaderGUI.cs

```diff
@@ -110,9 +111,10 @@ protected static class Styles
             public static readonly GUIContent enableSSAA = new GUIContent("Super Sample Anti-Aliasing", "Enable Super Sample Anti-Aliasing, a technique improves texture clarity at long distances");
             public static readonly GUIContent mipmapBias = new GUIContent("Mipmap Bias", "Degree to bias the mip map. A larger negative value reduces aliasing and improves clarity, but may decrease performance");
             public static readonly GUIContent lightMode = new GUIContent("Light Mode", "What Type of Direct Light Affects the Surface");
-            public static readonly string[] lightModeNames = new string[] { "Unlit", "Lit - Directional", "Lit - Distant" };
+            public static readonly string[] lightModeNames = new string[] { "Unlit", "Lit - Directional", "Lit - Distant" , "Non - Photorealistic" };

```

Comment:
I love that you put this option in a drop down so that the UI doesn't become cluttered with too many checkboxes. But there might be scenarios where people want to use [DistantLights](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/pull/37) with Non-Photorealistic rendering?

If that is the case maybe we move this property down to a separate check box that gets displayed when "Lit - Directional" or "Lit - Distant" is selected. Just like how specular highlights and spherical harmonics are toggled: 
![image](https://user-images.githubusercontent.com/13305729/195491970-5ac99ca9-ce3d-45a2-898d-3ec31969bc72.png)